rocketpy/sensitivity/sensivity_model.py

```diff
@@ -0,0 +1,515 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import norm
+from ..tools import check_requirement_version, import_optional_dependency
+
+
+class SensitivityModel:
+    """Performs a 'local variance based first-order
+    sensitivity analysis' considering independent input parameters.
+
+    The core reference for global variance based sensitivity analysis is
+    [1]. Our method implements a local version that only considers first
+    order terms, which correspond to linear terms. Albeit the flight
+    function is nonlinear, the linear hypothesis might be adequate when
+    performing *local* sensitivity analysis.
+
+    The model is fit using separate multiple linear regression for each
+    target variable passed and using the parameters as covariates.
+
+    References
+    ----------
+    [1] Sobol, Ilya M. "Global sensitivity indices for nonlinear mathematical models and their Monte Carlo estimates." Mathematics and computers in simulation 55.1-3 (2001): 271-280.
+    """
+
+    def __init__(
+        self,
+        parameters_names,
+        target_variables_names,
+    ):
+        self.__check_requirements()
+        self.n_parameters = len(parameters_names)
+        self.parameters_names = parameters_names
+        self.parameters_info = {
+            parameter: {
+                "nominal_mean": None,
+                "nominal_sd": None,
+            }
+            for parameter in parameters_names
+        }
+
+        self.n_target_variables = len(target_variables_names)
+        self.target_variables_names = target_variables_names
+        self.target_variables_info = {
+            variable: {
+                "nominal_value": None,
+                "sd": None,
+                "var": None,
+                "model": None,
+                "sensitivity": {parameter: None for parameter in self.parameters_names},
+                "LAE": None,  # Linear Approximation Error
+            }
+            for variable in target_variables_names
+        }
+
+        self.number_of_samples = None
+
+        # Flags for nominal parameters
+        self._nominal_parameters_passed = False
+        self._nominal_target_passed = False
+
+        self._fitted = False
+
+        return
+
+    def set_parameters_nominal(
+        self,
+        parameters_nominal_mean,
+        parameters_nominal_sd,
+    ):
+        """Set parameters nominal mean and standard deviation
+
+        Parameters
+        ----------
+        parameters_nominal_mean : np.array
+            An array contaning the nominal mean for parameters in the
+            order specified in parameters names at initialization
+        parameters_nominal_sd : np.array
+            An array contaning the nominal standard deviation for
+            parameters in the order specified in parameters names at
+            initialization
+        """
```

Comment:
So do you have to set the mean **and** sd simultaneously? 

Also, do you have to set mean and sd of all parameters? Setting for just some of them does not work?

Another thing, to run a Monte Carlo sim, the mean and sd is already given in the Monte Carlo class right? So it would be natural to get them from there automatically

Reply:
It would be great to recover the mean and sd used in the simulation! I tried to find where it was stored, but I might have missed it. Do you know where it is stored?

Reply:
Indeed, you don't have to provide both, thought that, if you had one of them, you probably had the other.

Reply:
I think we could use another PR to better integrate the `MonteCarlo` and the `SensitivityModel` classes.